subprojects/packagefiles/tinycdb/LICENSE.build

```diff
@@ -0,0 +1,502 @@
+                  GNU LESSER GENERAL PUBLIC LICENSE
```

Comment:
absolutely not.

Reply:
I have no idea what you mean by this comment.

Reply:
I am *guessing* it is an objection to your choice of license... ?